frontend/src/app/components/selectors/SelectorBase.ts

```diff
@@ -14,32 +14,39 @@ export abstract class SelectorBase<CustomSliceSelection extends SliceSelection>
 
     public stateChange: EventEmitter<SelectionStateMessage>;
 
+    public isOnlyOneSelectionPerSlice(): boolean {
```

Comment:
Good job creating this one as method :) . Now we need to discuss if we want to have many selection on one slice or not.